wiring/src/spark_wiring_stream.cpp

```diff
@@ -192,7 +193,7 @@ float Stream::parseFloat(char skipChar){
 
   c = peekNextDigit();
     // ignore non numeric leading characters
-  if(c < 0)
+  if(!std::isprint(c))
```

Comment:
I suggest instead of using `isprint()` here we store the result of the `peekNextDigit()` call in a separate `int` variable and assign it to `c` if it's non-negative.

Reply:
:+1:

Reply:
Fixed